infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java

```diff
@@ -78,6 +78,26 @@ public void warnBellatrixParameterChanged(final String parameterName, final Stri
             Color.YELLOW));
   }
 
+  public void warnDenebEpochsStoreBlobsParameterSet(
+      final String epochsStoreBlobs, final boolean isOverriden, final String specValue) {
+    if (isOverriden) {
+      final boolean isMax = String.valueOf(Integer.MAX_VALUE).equals(epochsStoreBlobs);
```

Comment:
since you are using `Integer.MAX_VALUE` here again. I'd suggest tu put it as a constant like `final static Integer MAX_EPOCHS_STORE_BLOBS = Integer.MAX;` and use it everywhere.